src/ort_genai.h

```diff
@@ -4,7 +4,7 @@
 #pragma once
 
 #include <memory>
-#include <span>
+#include "span.h"  // TODO Use the std span header if we can assume C++20.
```

Comment:
Hmm, this will break our release package as we don't ship span.h with this file. Baiju has another PR where we put the span functions inside an `#if __cplusplus >= 202002L`

https://github.com/microsoft/onnxruntime-genai/pull/239

What do you think?